### PR TITLE
Feature/new checkbox icons

### PR DESCRIPTION
**Pull request by ccedrone**
Ensuring I have the states and icons replaced where appropriate.

### CODE REVIEW

File: packages/react-magma-dom/src/components/IndeterminateCheckbox/index.tsx [hunk 1]
```diff
@@ -6,12 +6,16 @@ import {
...
+  CheckBoxIcon,
+  CheckBoxOutlineBlankIcon,
+  IndeterminateCheckBoxIcon,
+} from 'react-magma-icons';
 import { FormGroupContext } from '../FormGroup';
 import { InputMessage } from '../Input/InputMessage';
 import { StyledLabel } from '../SelectionControls/StyledLabel';
 import { StyledContainer } from '../SelectionControls/StyledContainer';
-import styled from '@emotion/styled';
+// import styled from '@emotion/styled';
```
Comment (joasmi723) [inline packages/react-magma-dom/src/components/IndeterminateCheckbox/index.tsx:18]: Please remove the commented line

File: packages/react-magma-dom/src/components/Radio/index.tsx [hunk 2]
```diff
@@ -80,26 +105,13 @@ const StyledFakeInput = styled.span<{
...
+  color: ${props => buildRadioIconColor(props)};
   border-radius: 100%;
-  box-shadow: ${props =>
-    props.isInverse && props.hasError
-      ? `0 0 0 1px ${props.theme.colors.neutral08}`
-      : '0 0 0'};
   cursor: ${props => (props.disabled ? 'not-allowed' : 'pointer')};
   margin: ${props =>
-    props.textPosition === 'left' ? '2px 0 0 10px' : '2px 10px 0 0'};
+    props.textPosition === 'left' ? '0 0 0 10px' : '0 10px 0 0'};
```
Comment (joasmi723) [inline packages/react-magma-dom/src/components/Radio/index.tsx:112]: Please replace hardcoded pixel values with values from the theme.

File: packages/react-magma-dom/src/components/SelectionControls/InputStyles.tsx [hunk 3]
```diff
@@ -2,14 +2,13 @@ import { css } from '@emotion/core';
 
 export const DisplayInputStyles = css`
   align-items: center;
-  border: 2px solid;
   display: flex;
-  height: 20px;
+  height: 24px;
```
Comment (joasmi723) [inline packages/react-magma-dom/src/components/SelectionControls/InputStyles.tsx:6]: Please replace hardcoded pixel values with values from the theme.

File: packages/react-magma-dom/src/components/SelectionControls/InputStyles.tsx [hunk 4]
```diff
@@ -22,10 +21,10 @@ export const DisplayInputStyles = css`
     // active state
```
Comment (joasmi723) [inline packages/react-magma-dom/src/components/SelectionControls/InputStyles.tsx:21]: Same here - whenever possible, use the px values from the theme instead of hardcoded ones.

File: website/react-magma-docs/src/pages/api/indeterminate.mdx [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+title: Indeterminate Checkbox
+props:
+  - IndeterminateCheckboxProps
+---
+
+<DocsHeading to="/design/selection-controls" type="api">
+  Indeterminate
+</DocsHeading>
+
+The `IndeterminateCheckbox` component is similar to the `Checkbox` component, except that instead of accepting a boolean `checked` value,
```
Comment (joasmi723) [inline website/react-magma-docs/src/pages/api/indeterminate.mdx:12]: Let's make the words "Checkbox component" a link to the checkbox page of the docs.

File: website/react-magma-docs/src/pages/api/checkboxes.mdx [hunk 6]
```diff
@@ -80,77 +80,6 @@ export function Example() {
 }
 ```
 
-## Indeterminate
```
Comment (joasmi723) [inline website/react-magma-docs/src/pages/api/checkboxes.mdx:83]: Because we're removing this section of the docs on this page, I think we should have a link to to the Indeterminate Checkbox docs page on this page.  Can you add a new sentence with a link to IndeterminateCheckbox  up at the top near where we also link to the radio buttons etc?

File: packages/react-magma-dom/src/components/Checkbox/Checkbox.stories.tsx [hunk 7]
```diff
@@ -15,69 +12,135 @@ export default {
...
+          />
+        </FormGroup>
+        <FormGroup labelText="Colors">
+          <Checkbox
+            isInverse
+            checked
+            color={magma.colors.primary}
+            labelText="Primary checked"
+          />
+          <Checkbox
```
Comment (joasmi723) [inline packages/react-magma-dom/src/components/Checkbox/Checkbox.stories.tsx:103]: If we aren't going to support the `color` prop on inverse checkboxes by always making them white, we don't need multiple examples of inverse checkboxes with different colors.  The same would go for the mdx page in the docs.

File: packages/react-magma-dom/src/components/Dropdown/DropdownSplitButton.tsx [hunk 8]
```diff
@@ -44,9 +44,9 @@ export const DropdownSplitButton = React.forwardRef<
 
   const buttonIcon =
     context.dropDirection === DropdownDropDirection.up ? (
-      <ArrowDropUpIcon size={10} testId="caretUp" />
+      <ArrowDropUpIcon size={24} testId="caretUp" />
```
Comment (joasmi723) [inline packages/react-magma-dom/src/components/Dropdown/DropdownSplitButton.tsx:47]: I know you're going to do more changes to the split button in another PR, but we can get the sizes from the theme -- `theme.iconSizes.medium`

**Comment by orion-cengage** [comment]
I think you want one of these examples to be selected and disabled, right?

image

**Review by joasmi723 — APPROVED** [review]
